### DISCUSSION TITLE
Completely fully customizable theme

### DISCUSSION THREAD

**Discussion by Onebeld** [AvaloniaUI/Avalonia#5704] [General]
Hello everyone!
I've been working on building a library for the Avalonia user interface lately and I'm eager to show you. The main function is to dynamically change the theme. This library is already available in [NuGet](https://www.nuget.org/packages/OlibUI/).
I also want to know your opinion: what to add? what to remove? what to change?

![Dfzl13GcYr](https://user-images.githubusercontent.com/44552715/112000579-f08e7300-8b3f-11eb-9247-a23e9483a9fc.gif)

**Comment by maxkatz6** [comment]
Awesome!
Thanks for contributing into Avalonia community projects :)
I believe you should add your project to the Awesome Avalonia repository, so user will be able to find it easier:
https://github.com/AvaloniaCommunity/awesome-avalonia
